packages/pn-commons/src/components/CustomTooltip.tsx

```diff
@@ -7,12 +8,12 @@ import Tooltip from '@mui/material/Tooltip';
 type Props = {
   tooltipContent: any;
   openOnClick: boolean;
-  children: ReactElement<any, any>;
   sx?: SxProps;
   onOpen?: () => void;
+  children: JSX.Element;
```

Comment:
Because of cloneElement which doesn't accept ReactNode so I modified the children type and removed the "children as JSX.Element" in cloneElement param by simply replacing with children.